nautobot/dcim/tests/test_views.py

```diff
@@ -646,24 +646,27 @@ class ManufacturerTestCase(ViewTestCases.OrganizationalObjectViewTestCase):
     @classmethod
     def setUpTestData(cls):
 
-        Manufacturer.objects.create(name="Manufacturer 1", slug="manufacturer-1")
-        Manufacturer.objects.create(name="Manufacturer 2", slug="manufacturer-2")
-        Manufacturer.objects.create(name="Manufacturer 3", slug="manufacturer-3")
-        Manufacturer.objects.create(name="Manufacturer 8")
+        manufacturer = Manufacturer.objects.first()
+
+        # FIXME(jathan): This has to be replaced with# `get_deletable_object` and
+        # `get_deletable_object_pks` but this is a workaround just so all of these objects are
+        # deletable for now.
+        DeviceType.objects.all().delete()
```

Comment:
Won't this be a problem as soon as we add Device factories, since `Device.device_type` uses `models.PROTECT`?

Reply:
Yes! Thus the comments! This is really just to get this merged in so we don't keep kicking the merge conflicts can down the road.